text/0020-distingush_manual_vs_scripted_transforms.md

```diff
@@ -0,0 +1,312 @@
+
+```
+$ qri get transform me/ca_prime_ministers
+null
+```
+
+Also, `qri update` now gives us a new error:
+
+```
+$ qri update me/ca_prime_ministers
+error: no transform script in most recent dataset. There is a transform script 2 commits back that adjusts:
+  body
+
+to run an update using the most recent transform, run:
+  qri update --recall-tf me/ca_prime_ministers
+```
+
+This missing transform is vital for reproducibility reasons, the missing transform indicates that no transform script was executed to get from the snapshot that had the old meta to the new meta.
+
+Resurrecting the transform is relatively easy, we follow the instructions from the error:
+```
+$ qri update --recall-tf me/ca_prime_ministers
+🤖 executing transform
+✅ transform complete
+updated dataset b5/ca_prime_minsters
+```
+
+Finally, if we want to both re-run the existing transform _and_ change things about a dataset in a single commit, we can still do that so long as the transform doesn't affect any fields we're trying to manually change by running `qri save with --recall-tf`. In this example we'll adjust the schema to add more specificity, so we adjust dataset.yaml:
+
+`dataset.yaml`:
+```yaml
+structure:
+  format: json
+  schema:
+    type: array
+    items:
+      type: object
+      properties:
+        name:
+          type: string
+        description:
+          type: string
+          maxLength: 140
+```
+
+And run save with `--recall-tf`:
+
+```
+$ qri save --file=dataset.yaml --recall-tf me/ca_prime_ministers
+🤖 executing transform
+✅ transform complete
+updated dataset b5/ca_prime_minsters
+```
+
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+## Human and Machine Transforms that affect the same field is now an error
+
+While this RFC places the definition at the field level, our initial implemenation will operate at the component level for simplicity.
+
+This does mean we'll need tools to check which components & fields of a dataset a transform affects. Thanks to the way starlark works (and our implementation), this'll be pretty simple.
+
+Ways to resolve the error:
+* adjust your commit so that it doesn't affect
+* `--drop-transform` will remove the transform moving forward
+
+## The job of a transform script is to generate the next state snapshot from the existing snapshot
+This means the dataset passed into `transform(ds,ctx)` will be the existing dataset in Qri. Because human and machine transforms are mutually exclusive, script authors only need to consider which fields they wish to remain editable by users.
+
+
+## `save` is for humans, `update` is for machines
+Save is now the more powerful of the two commands. 
+
+By default save will _not_ recall and re-run transforms, but will run provided transforms. running `qri save` without providing a tra
+
+Local updates will need some refinement. Running `qri update me/dataset` will now only succeed if a 
+Update gets a new flag: `--recall-tf`
+
```

Comment:
Please add an answer for a question raised in the earlier feedback: when does the body get validated against the schema? I believe it's only after the transforms are all applied, but that should be explicitly stated here, since it gets brought up at the start of the RFC.